crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/MaatCourtDataServiceTest.java

```diff
@@ -60,7 +55,7 @@ void givenUpdateRequest_whenPersistMeansAssessmentIsInvoked_thenPutRequestIsSent
     @Test
     void givenRepId_whenGetPassportAssessmentFromRepIdIsInvoked_thenResponseIsReturned() {
         PassportAssessmentDTO expected = new PassportAssessmentDTO();
-        when(maatAPIClient.get(any(), anyString(), any()))
+        when(maatAPIClient.getPassportAssessmentFromRepId( any()))
```

Comment:
Could be good to improve some of these tests and specify the types rather than just using `any()`